charts/seed-monitoring/charts/prometheus/templates/config.yaml

```diff
@@ -472,3 +472,40 @@ data:
       - target_label: __address__
         replacement: 127.0.0.1:9115
         action: replace
+
+{{- range $i := until 10 }}
```

Comment:
Why do we need 10 such jobs?

Reply:
@marwinski requested that multiple requests be sent to the API Server to observe the behaviour. Multiple jobs for the blackbox exporter is the easiest solution.